src/widgets/widget_definitions.rs

```diff
@@ -146,6 +147,29 @@ pub(super) fn resolve_orientable_attrs(bargs: &mut BuilderArgs, gtk_widget: &gtk
 
 // concrete widgets
 
+/// @widget combo-box
+fn build_gtk_combo_box(bargs: &mut BuilderArgs) -> Result<gtk::ComboBoxText> {
+    let gtk_widget = gtk::ComboBoxText::new();
+    let on_change_handler_id: Rc<RefCell<Option<glib::SignalHandlerId>>> = Rc::new(RefCell::new(None));
+    resolve_block!(bargs, gtk_widget, {
+        // @prop items
+        prop(items: as_vec) {
+            for i in items {
+            gtk_widget.append_text(&i);
+            }
+        },
```

Comment:
This will not correctly handle `items` being changed. currently, it would just add elements to the end, rather than replacing the contents. you'll need to reset before appending, first.

Reply:
fixed